cuda/apps/src/kinect_ransac.cpp

```diff
@@ -99,23 +99,22 @@ class SimpleKinectTool
         else
         {
           typename SampleConsensusModel1PointPlane<Storage>::IndicesPtr inliers_stencil;
-          inliers_stencil = sac.getInliersStencil ();
```

Comment:
It might not be needed to even declare the `inliers_stencil` variable => this branch isn't needed.